content/chapters/app-interact/lab/content/time-server.md

```diff
@@ -10,15 +10,15 @@ The client will then print the received time to the console.
 Let's build and run this example
 
 ```
-student@os:~/.../lab/time-server$ make
+student@os:~/.../support/time-server$ make
 gcc -Wall -o server server.c
 gcc -Wall -o client client.c
-student@os:~/.../lab/time-server$ ./server
+student@os:~/.../support/time-server$ ./server 2000
```

Comment:
The server doesn't use the command line arguments (like `2000`).

Reply:
Commited an unintended change.